### ISSUE TITLE
Netcat: -c tries to start /bin/sh

### ISSUE THREAD

**Issue by Neo-Oli** [termux/termux-packages#777]
I've been trying to start a simple webserver with netcat, but the `-c` option requires `/bin/sh`

```
while true ; do netcat -l -p 10000 -c 'echo -e "HTTP/1.1 200 OK\n\n $(date)"'; done
Ncat: /bin/sh is not executable, so `-c' won't work. QUITTING.
```

**Comment by fornwall** [comment]
Thanks, this should now be fixed in version `7.40-1` of the updated netcat package.

As a side note: Note that `echo -e` is not portable across shells. The built in `echo` command in the dash shell which Termux, Ubuntu and others use as `sh` does not interpret `-e` - use `printf` for if you need escape sequences portably across shells.

**Comment by Neo-Oli** [comment]
Thanks
